rapid7_insight_agent/plugin.spec.yaml

```diff
@@ -0,0 +1,191 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: rapid7_insight_agent
+title: Rapid7 Insight Agent
+description: Take actions on an Insight Agent
+version: 1.0.0
+vendor: rapid7
+support: rapid7
+status: []
+resources:
+  source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/rapid7_insight_agent
+  license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+tags:
+- rapid7
+- agent
+- insight
+- edr
+- endpoint
+- detection
+- response
+- idr
+- ivm
+hub_tags:
+  use_cases: [threat_detection_and_response]
+  keywords: [rapid7, agent, endpoint, response, detection, insight, idr, ivm]
+  features: []
+types:
+  attributes:
+    key:
+      title: "Key"
+      type: string
+      description: "Key"
+      required: false
+    value:
+      title: "Value"
+      type: string
+      description: "Value"
+      required: false
+  hostNames:
+    name:
+      title: "Name"
+      type: string
+      description: "Name"
+      required: false
+  primaryAddress:
+    ip:
+      title: "IP"
+      type: string
+      description: "IP"
+      required: false
+    mac:
+      title: "MAC"
+      type: string
+      description: "MAC address"
+      required: false
+  uniqueIdentity:
+    id:
+      title: "ID"
+      type: string
+      description: "ID"
+      required: false
+    source:
+      title: "Source"
+      type: string
+      description: "Source"
+      required: false
+  agent:
+    attributes:
+      title: "Attributes"
+      type: "[]attributes"
+      description: "Attributes"
+      required: false
+    description:
+      title: "Description"
+      type: string
+      description: "Description"
+      required: false
+    hostNames:
+      title: "Host Names"
+      type: "[]hostNames"
```

Comment:
```suggestion
      type: "[]hostName"
```

Reply:
Can't change this, it's how it pops out in the JSON.